modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -215,7 +215,9 @@ public function enqueue_scripts() {
 			'wpcomGutenberg',
 			array(
 				'switchToClassic' => array(
-					'isVisible' => false,
+					'isVisible' => $is_calypsoify,
```

Comment:
Note that you can also use here `$this->is_iframed_block_editor()` which feels more semantic? We only want to show the "Switch to classic" button on the iframed block editor.

Reply:
b0a0c3f40debc8b668cbd3bd75a339929a88fbed